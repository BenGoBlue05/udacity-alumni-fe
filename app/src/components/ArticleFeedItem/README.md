## ArticleFeedItem Component
A component that displays information from a single article.

### Example

```js
<ArticleFeedItem article={article} />
```

### Props

| Prop          | Type     | Default     | Possible Values
| ------------- | -------- | ----------- | ---------------------------------------------
| **article**    | Object   |             | Accepts an object which includes data from a single article.
