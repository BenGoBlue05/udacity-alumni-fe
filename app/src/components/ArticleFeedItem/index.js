import React, { PropTypes } from 'react';

import styles from './index.module.scss';
import cssModules from 'react-css-modules';

const ArticleFeedItem = (props) => (
  <div className={styles.articleFeedItem}>
  </div>
);

ArticleFeedItem.propTypes = {

};

export default cssModules(ArticleFeedItem, styles);
