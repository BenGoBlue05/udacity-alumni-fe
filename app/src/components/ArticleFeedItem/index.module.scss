.readMoreWrapper {
  flex: 1;
  margin-top: 24px;
  margin-bottom: 24px;
}

.articleDate {
  display: flex;
  align-items: center;
  justify-content: center;
  margin-bottom: 40px;
  @media screen and (min-width: 768px) {
    left: 31em;
    top: 3px;
    z-index: 2;
    position: absolute;
    transform: translate(100%, 0);
    margin-left: 0px;
  }
}

.articleFeedItem {
  p {
    font-family: 'Lato';
    font-size: 18px !important;
    line-height: 1.58 !important;
    letter-spacing: -.004em !important;
    font-weight: 400 !important;
  }
  h2 {
    font-weight: 700;
    font-style: normal;
    font-size: 40px;
    margin-left: -2.5px;
    line-height: 1.04;
    letter-spacing: -.028em;
  }
}
