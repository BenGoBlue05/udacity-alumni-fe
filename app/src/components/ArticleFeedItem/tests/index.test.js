import ArticleFeedItem from '../index';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import React from 'react';

describe('<ArticleFeedItem />', () => {
  it('should render with default props', () => {
    const wrapper = shallow(
      <ArticleFeedItem />
    );
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
