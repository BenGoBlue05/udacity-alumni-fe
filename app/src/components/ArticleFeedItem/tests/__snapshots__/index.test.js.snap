exports[`<ArticleFeedItem /> should render with default props 1`] = `<div />`;
